packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -3513,33 +3513,32 @@ void main() {
     Offset topLeft = editable.localToGlobal(
       editable.getLocalRectForCaret(const TextPosition(offset: 7)).topLeft,
     );
-    expect(topLeft.dx, equals(362));
+    expect(topLeft.dx, equals(469));
 
     topLeft = editable.localToGlobal(
       editable.getLocalRectForCaret(const TextPosition(offset: 8)).topLeft,
     );
-    // Caret is capped at text length.
```

Comment:
I was wrongly assume this when i first wrote the test.
This would have been true if container does not have a width constraint.